build.rs

```diff
@@ -40,5 +48,60 @@ fn main() {
         println!("cargo:rustc-link-lib=static=chck-xdg");
         println!("cargo:rustc-link-lib=static=wlc-protos");
         println!("cargo:rustc-link-lib=static=wlc");
+    } else if let Err(s) = find() {
+        let _ = writeln!(io::stderr(), "{}", s);
+        process::exit(1);
+    }
+}
+
+fn find() -> Result<(), Error> {
+
+    let package_name = "wlc";
+    let shared_libs = ["wlc"];
+
+    // https://github.com/way-cooler/rust-wlc#rust-wlc currently
+    // (i.e. commit 97e0d5816fe7ec97dae20196d1e25c550f60f08f) suggests
+    // version 0.0.7.
+    let version = if cfg!(feature = "0.0.8") {
+        "0.0.8"
+    } else if cfg!(feature = "0.0.7") {
+        "0.0.7"
+    } else if cfg!(feature = "0.0.6") {
+        "0.0.6"
+    } else {
+        "0.0.5"
+    };
+
+    // Is this Windoze-specific? Do we need to carry this along on Linux with
+    // Wayland at all?
+    let target = env::var("TARGET").unwrap();
+    let hardcode_shared_libs = target.contains("windows");
```

Comment:
We shouldn't need this, as obviously Wayland isn't on Windows ;P. I tested it without it and it runs fine on my machine, so it's probably safe to remove.